src/subdataframe/subdataframe.jl

```diff
@@ -158,11 +164,11 @@ end
 ##
 ##############################################################################
 
-Base.copy(sdf::SubDataFrame) = parent(sdf)[rows(sdf), :]
-
-without(sdf::SubDataFrame, c) = view(without(parent(sdf), c), rows(sdf), :)
-# Resolve a method ambiguity
-without(sdf::SubDataFrame, c::Vector{<:Integer}) = view(without(parent(sdf), c), rows(sdf), :)
+Base.copy(sdf::SubDataFrame) = parent(sdf)[rows(sdf), parentcols(sdf, :)]
 
+function without(df::SubDataFrame, icols::Vector{<:Integer})
```

Comment:
```suggestion
function without(df::SubDataFrame, icols::AbstractVector{<:Integer})
```
Couldn't this method be common to all `AbstractDataFrame` types?

Reply:
It can (in fact - I initially removed it, but then mistakengly thought that `sdf[cols]` loses a view - but it returns `SubDataFrame` now so all should be correct).